### PR TITLE
Remove matplotlib references

### PR DESCRIPTION
**Pull request by montezdesousa**
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/futures/databento_view.py [hunk 1]
```diff
@@ -44,61 +43,23 @@ def display_historical(
...
-        if has_volume:
-            candle_chart_kwargs["volume"] = external_axes[1]
-        mpf.plot(data, **candle_chart_kwargs)
+    data = stocks_helper.process_candle(data)
+    figure = stocks_helper.display_candle(
+        symbol=symbol,
+        data=data,
+        external_axes=external_axes,
+        raw=raw,
+    )
```
Comment (tehcoderer) [inline openbb_terminal/futures/databento_view.py:62]: Here I would call and return it like this😊

```python
    data = stocks_helper.process_candle(data)
    figure = stocks_helper.display_candle(
        symbol=symbol,
        data=data,
        external_axes=True,
        raw=raw,
    )
    return figure.show(external=external_axes)
   ```

**Review by tehcoderer — COMMENTED** [review]
Here from 
```python
from openbb_terminal import OpenBBFigure
from openbb_terminal.core.plots.plotly_helper import theme
```

You can do this
```python
from openbb_terminal import OpenBBFigure, theme
```

File: openbb_terminal/futures/databento_view.py (hunk 1 above)
Comment (montezdesousa) [reply openbb_terminal/futures/databento_view.py:62]: done, thanks!

**Review by tehcoderer — APPROVED** [review]
Looks good!!! Lets gooooo 🚀🚀🚀

**Pull request merged**